plugins/out_cloudwatch_logs/cloudwatch_api.c

```diff
@@ -362,6 +523,60 @@ static int truncate_log(const struct flb_cloudwatch *ctx, const char *log_buffer
     return FLB_FALSE;
 }
 
+// Helper function to remove keys from a nested map
```

Comment:
Can we update this comment to specify that we're specifically trying to remove the AWS_ENTITY_PREFIX keys?

Reply:
addressed in new commit